### PR TITLE
in review trblshoot

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: clusters/support_troubleshooting/trouble_network_config_bm.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ 
+[#symptom-worker-node-fail]
+== Symptom: Installation for day two workers fails
+
+At the Discover phase, the host cannot configure the network. Check for the following symptoms and messages:
+----
+This host is pending user action. Host timed out when pulling ignition. Check the host console...
+----
+
+Also check all the MCs (we need this actual term) from the `/sysroot/etc/NetworkManager/system-connections/` or `/sysroot/etc/sysconfig/network-scripts/` file paths.
```
Comment (swopebe) [inline clusters/support_troubleshooting/trouble_network_config_bm.adoc]: I need MC term here.

File: clusters/support_troubleshooting/trouble_network_config_bm.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+[#resolving-cluster-rotating-agents]
+== Resolving the problem: Manually configure the network
+
+Perform the following task to manually configure your network:
+
+. Delete the node from your hub cluster.
+. Create your cluster from the console again.
+. Add or append the following annotation to the BMH of the new node:
+
```
Comment (swopebe) [inline clusters/support_troubleshooting/trouble_network_config_bm.adoc:40]: Need BMH

**Comment by jgato** [comment]
I will create a PR (to your branch) to help with the content, to make it similar to the previous analyses we already did. I hope it will help. 
It will include also the clarifications requested above.

**Comment by jgato** [comment]
everything looks good to me

**Review by jc-berger — COMMENTED** [review]
/lgtm

**Pull request merged**